### PR TITLE
New eth bidsubmit

### PR DESCRIPTION
**Pull request by changchaishi**
replace of #36 , new bid submit should merge to branch `dr-upload-revert`

### CODE REVIEW

File: pt/blockchain/trigger.py [hunk 1]
```diff
@@ -35,6 +36,49 @@ def bidsubmit():
...
+                volumes,
+                prices
+            )
+            logger.info(f"Contract Transaction Result: {result}")
+
+            if result:
+                tx_hash = result[0]["transactionHash"]
+                for bid in bidsubmits:
+                    bid.status = "已投標"
+                    bid.win: 0
```
Comment (ttw225) [inline pt/blockchain/trigger.py]: 冒號！

**Review by ttw225 — CHANGES_REQUESTED** [review]
冒號！

File: pt/blockchain/trigger.py (hunk 1 above)
Comment (changchaishi) [reply pt/blockchain/trigger.py]: Thanks, its fixed by 5298eedac499a8dd305925c3af7ba9df00e43a6e

**Review by ttw225 — APPROVED** [review]
LGTM~

**Pull request merged**